iothub/service/src/JobProperties.cs

```diff
@@ -118,6 +118,13 @@ public JobProperties()
         [JsonProperty(PropertyName = "storageAuthenticationType", NullValueHandling = NullValueHandling.Ignore)]
         public StorageAuthenticationType? StorageAuthenticationType { get; set; }
 
+        /// <summary>
+        /// The managed identity used to access the storage account for import and export jobs.
```

Comment:
@nimengan - Here is the description I added. We should keep this consistent across SDKs and in the swagger. At the moment we don't have any description for this in the swagger.